doc/pages/faces.asciidoc

```diff
@@ -2,183 +2,188 @@
 
 == Declaration
 
-A 'face' determines how text is displayed, it has a foreground
-color, a background color, and some attributes. The value of a face has the
+A *face* determines how text is displayed. It has a foreground color,
+a background color, and some attributes. The value of a face has the
 following format:
 
 -----------------------------------------------------------
 [fg_color][,bg_color[,underline_color]][+attributes][@base]
 -----------------------------------------------------------
 
-'fg_color', 'bg_color', 'underline_color'::
-    a color whose value can be expressed in the following formats:
-        *black*, *red*, *green*, *yellow*, *blue*, *magenta*, *cyan*, *white*:::
-        *bright-black*, *bright-red*, *bright-green*, *bright-yellow*:::
-        *bright-blue*, *bright-magenta*, *bright-cyan*, *bright-white*:::
-            a named color
-        *default*:::
-            keep the existing color
-        *rgb:RRGGBB*:::
-            hexadecimal value
-        *rgba:RRGGBBAA*:::
-            hexadecimal value
-    if unspecified or empty, *default* is used.
-    Alpha values are used to blend the face onto either its base or the existing
-    color. For technical reasons alpha values must be >16 if specified.
-
-'attributes'::
-    string whose individual letters set an attribute:
+fg_color, bg_color, underline_color::
+    A color whose value can be:
+        A named color:::
+            *black*, *red*, *green*, *yellow*, *blue*, *magenta*, *cyan*,
+            *white* *bright-black*, *bright-red*, *bright-green*,
+            *bright-yellow* *bright-blue*, *bright-magenta*, *bright-cyan*,
+            *bright-white*
+       The existing color:::
+            *default*
+        A hexadecimal value:::
+            *rgb:RRGGBB*, *rgba:RRGGBBAA*
+    If left unspecified, the value of *default* is used.
+    Alpha values are used to blend the face onto either its base (see below) or
+    else onto whatever color happens to be used at the moment. For technical
+    reasons, alpha values must be >16.
+
+attributes::
+    A string whose individual letters each set an attribute:
         *u*:::
-            underline
+            Underline
         *c*:::
-            curly underline, takes precedence over underline if both are
-            specified
+            Curly underline
+            Note: This takes precedence over underline if both are specified.
         *r*:::
-            reverse
+            Reverse
         *b*:::
-            bold
+            Bold
         *B*:::
-            blink
+            Blink
         *d*:::
-            dim
+            Dim
         *i*:::
-            italic
+            Italic
         *s*:::
-            strikethrough
+            Strikethrough
```

Comment:
I don't think we should capitalize this, it looks worse IMHO

Reply:
Okay, after checking with the OED and [MW](https://www.merriam-webster.com/dictionary/editor), I agree with you that the definitions don't need to be capitalized or punctuated at the end. I'll revert these changes.

Reply:
We can continue this line of thought in the other comment thread below.